### DISCUSSION TITLE
Unofficial API documentation available for the HA community - based on 5MP cameras

### DISCUSSION THREAD

**Discussion by mnpg** [fwestenberg/reolink_dev#448] [Show and tell]
hi everyone,

i share with you my unofficial API documentation for the HA community to contribute with my little stones in the development of the reolink_dev addon.
All the infos is based on my own cameras (5MP models from the E1 series and 2 RLC-5xx). 

Right now, you'll find one for the non-ai cameras and the second contains the first one + commands for AI.

Most commands are detailled (AI, non AI, PTZ or not, Zoom or not,...)  **but NOT ALL the commands** (check the official one)
Also recent ones, taken from the latest firmware of january 2022, are added.

These documentations are in French (sorry guys, i ain't got time to translate it, use google translate)

here the link : 
https://github.com/mnpg/Reolink_api_documentations > folder unofficial

hope it will help you

Regards

**Comment by cpainchaud** [comment]
I had a quick look and it's quite extensive. Indeed people is going to complain it's in English :)

**Reply by mnpg** [reply]
merci Christophe

**Reply by mnpg** [reply]
on peut se parler français entre nous :)
